### PR TITLE
Form 4192: Ability to download Form 4192#14080

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
As a Veteran submitting a claim for Individual Unemployability on the Vets.gov platform, I need the ability to download Form 21-4192 so that I can then complete it and upload it.

## Testing done
- [x] local
- [x] unit

## Screenshots
screen shot 2019-01-15 at 1 39 46 pm

## Acceptance criteria
- [x] The user has the ability to click on a hyperlink for Form 4192 and download it
- [x] The hyperlink should take the user to the pdf version of form and allow for download of the form

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/config/4192/index.js [hunk 1]
```diff
@@ -1,18 +1,50 @@
...
       pastEmploymentFormIntro: {
         path: 'past-employment-walkthrough-choice',
         uiSchema: pastEmploymentFormIntro.uiSchema,
         schema: pastEmploymentFormIntro.schema,
       },
+      // Form Tutorial (multiple pages)
+      // Download
+      pastEmploymentFormDownload: {
+        path: 'past-employment-download',
+        depends: formData => isDownloading(formData),
```
Comment (dmccurdybah) [inline src/applications/disability-benefits/all-claims/config/4192/index.js]: This could just be `depends: isDownloading,`

**Review by devin-mccurdy — APPROVED** [review]

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience? Thank you!

File: src/applications/disability-benefits/all-claims/config/4192/index.js [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+  pastEmploymentFormIntro,
+  pastEmploymentFormDownload,
+} from '../../pages';
 
 import { needsToEnterUnemployability } from '../../utils';
 
+// const showFormTutorial = formData => _.get(formData, 'view:upload4192Choice.view:4192Info', false);
+const isDownloading = formData =>
+  _.get(formData, 'view:upload4192Choice.view:download4192', false);
+// const isUploading = formData => _.get(formData, 'view:upload4192Choice.view:upload4192', false);
```
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/config/4192/index.js]: This should be removed from the commit if it's not used or a work in progress.

**Review by rianfowler — APPROVED** [review]
LGTM. just one minor nit

**Pull request merged**